cmd/statesync/trust.go

```diff
@@ -0,0 +1,157 @@
+package statesync
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/oasisprotocol/oasis-core/go/consensus/api"
+	"github.com/oasisprotocol/oasis-core/go/consensus/cometbft/config"
+
+	"github.com/oasisprotocol/cli/cmd/common"
+	cliConfig "github.com/oasisprotocol/cli/config"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
+	"github.com/spf13/cobra"
+)
+
+const (
+	// maxVerificationTime significantly overestimates the time needed for the
```

Comment:
This comment could be better structured, e.g. first description what this is, and then that it overestimates ...